### ISSUE TITLE
After upgrading to 6.3.0 getting "Error: EPERM: operation not permitted, unlink"

### ISSUE THREAD

**Issue by mikila85** [cypress-io/cypress#14644]
### Current behavior
After upgrading to 6.3.0 getting flaky errors on different tests, all the same:
```
11:50:05 EPERM: operation not permitted, unlink 'C:\Users\Administrator\AppData\Roaming\Cypress\cy\production\browsers\chrome-stable\run-2668\CypressCache\data_0'
11:50:05 Error: EPERM: operation not permitted, unlink 'C:\Users\Administrator\AppData\Roaming\Cypress\cy\production\browsers\chrome-stable\run-2668\CypressCache\data_0'
```

### Desired behavior
no errors

### Test code to reproduce
random

### Versions

last known working: 6.2.0

**Comment by jennifer-shehane** [comment]
Could you run Cypress in [debug mode](https://on.cypress.io/troubleshooting#Print-DEBUG-logs) mode and print the entire set of logs here?

**Comment by GregLahaye** [comment]
Also happening here on 6.3.0. Happens locally and on CI. Did not enable experimental studio option.

https://pastebin.com/2Yu1vdbi

Can email full set of log if needed.

**Comment by alapanme** [comment]
Also getting this error on Azure Pipeline with v6.3.0. Didn't faced this before. Also didn't enable the experimental studio option.

**Comment by PetMou** [comment]
This kind of error has occurred more in the past. (#3261)

**Comment by sofinapradhan** [comment]
I am getting this error too after the Cypress version is upgraded to 6.3.0. 
I get this error when multiple groups of specs are run in more than one terminals at once in the Cromer browser.

**Comment by GregLahaye** [comment]
This issue seems to be fixed in 6.4.0 for anyone wondering.

**Comment by jennifer-shehane** [comment]
Can everyone try to update to 6.4.0? We did revert a change from 6.3.0 in that release - although I'm not sure how it is related exactly.
